config/default.yml

```diff
@@ -237,6 +237,18 @@ Rails/EnvironmentComparison:
   Enabled: true
   VersionAdded: '0.52'
 
+Rails/EnvironmentVariableAccess:
+  Description: 'Do not access `ENV` directly after initialization.'
+  Enabled: pending
+  VersionAdded: '2.10'
```

Comment:
This was my best guess at what the version will be. 😬

Reply:
That's exactly right 😃